common/system.h

```diff
@@ -1181,10 +1181,13 @@ class OSystem : Common::NonCopyable {
 	 * and then manually compose whatever graphics we want to show in the overlay.
 	 * This works because we assume the game to be "paused" whenever an overlay
 	 * is active.
+	 *
+	 * @param inGame Whether the overlay is used to display GUI or in game images
+	 *
 	 */
 
 	/** Activate the overlay mode. */
-	virtual void showOverlay() = 0;
+	virtual void showOverlay(bool inGUI = true) = 0;
```

Comment:
It might be preferable to name the parameter `overlayCoordinates` or something similar to better reflect it's effect.

Reply:
Yes, I am not happy by the `inGUI` variable but `overlayCoordinates` is not better IMO as it doesn't say really more.
I will wait for other opinions on this variable name.

Reply:
while `inGUI` is not the best name, it is functional.

However, I would personally flip its value. E.g. GUI devs know that they're in GUI, so they could say `true`. Engine authors normally call GUI things or use overlay for something else. Thus, I'd make it `false` by default